CHANGELOG.md

```diff
@@ -14,7 +14,7 @@ X.Y.Z Release notes
 * [Sync] Exposed custom HTTP headers in `Realm.Configuration` (#1897).
 
 ### Bug fixes
-* None.
+* [RN Android] Ported workaround for crashes in `memmove`/`memcpy` on some old Android devices (#1163 and #1895).
```

Comment:
Perhaps just "Added workaround" ?

Reply:
Just trying to give credit to the great people who did the original workaround 😅